src/features/HomePage/MoviesSortingForm/MoviesSortingForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import CustomButton from '../../../components/CustomButton/CustomButton';
+import CustomInput from '../../../components/CustomInput/CustomInput';
+import { searchValueChanged, sortByLikes, sortByRating, resetSorting } from '../../../redux/movies/movies.actions';
+import styles from './MoviesSortingForm.module.scss';
+
+const MoviesSortingForm = ({ searchFieldValueChanged, sortByLikes, sortByRating, resetSorting }) => {
+  return (
```

Comment:
redundant `return` wording, again, please, pay more attention for such use cases and code style